src/vos/vos_tree.c

```diff
@@ -154,7 +154,9 @@ ktr_hkey_gen(struct btr_instance *tins, d_iov_t *key_iov, void *hkey)
 {
 	struct ktr_hkey		*kkey = (struct ktr_hkey *)hkey;
 	struct umem_pool        *umm_pool = tins->ti_umm.umm_pool;
+	struct vos_pool         *pool     = (struct vos_pool *)tins->ti_priv;
```

Comment:
I am not sure as far as i checked ti_priv might be inited as cont as well, could we make sure this function could be only used for pool?

Reply:
looks the ti_priv is vos_pool for this vos key tree.